src/checkstyle/Checker.hx

```diff
@@ -246,7 +250,23 @@ class Checker {
 		}
 	}
 
-	@SuppressWarnings("checkstyle:Dynamic")
+	function checkForExclude(moduleName:String):Bool {
+		if (excludes == null) return false;
+		var excludesForCheck:Array<String> = excludes.get(moduleName);
+		if (excludesForCheck == null || excludesForCheck.length == 0) return false;
+
+		var cls = file.name.substring(0, file.name.indexOf(".hx"));
```

Comment:
`.hx` is used earlier as well to push all the files into array
https://github.com/adireddy/haxe-checkstyle/blob/dev/src/checkstyle/Main.hx#L270

We can move the extensions into `config` and default it to `.hx`